packages/http-client-csharp/generator/TestProjects/CadlRanch/http/type/model/inheritance/single-discriminator/src/Generated/Models/Bird.cs

```diff
@@ -0,0 +1,30 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace _Type.Model.Inheritance.SingleDiscriminator.Models
+{
+    public abstract partial class Bird
+    {
+        protected Bird(string kind, int wingspan) => throw null;
```

Comment:
In autorest.csharp code, the base class `Bird` is not contain the `kind` (see following)
```
protected Bird(int wingspan)
{
        Wingspan = wingspan;
  }
```

Is the change in MGC for base class construct expected?